common/datetime.c

```diff
@@ -221,3 +223,39 @@ int32_t oe_datetime_compare(
 
     return 0;
 }
+
+oe_result_t oe_datetime_now(oe_datetime_t* value)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    time_t now;
+    struct tm* timeinfo;
+
+    if (value == NULL)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    time(&now);
+    timeinfo = gmtime(&now);
+
+    value->year = (uint32_t)timeinfo->tm_year + 1900;
+    value->month = (uint32_t)timeinfo->tm_mon + 1;
+    value->day = (uint32_t)timeinfo->tm_mday;
+    value->hours = (uint32_t)timeinfo->tm_hour;
+    value->minutes = (uint32_t)timeinfo->tm_min;
+    value->seconds = (uint32_t)timeinfo->tm_sec;
+
+    result = OE_OK;
+done:
+
+    return result;
+}
+
+void oe_datetime_log_info(const char* msg, const oe_datetime_t* date)
+{
+    if (oe_get_current_logging_level() >= OE_LOG_LEVEL_INFO)
+    {
```

Comment:
To avoid too much debug message in the verbose mode, 
do you want to consider only log datetime information in VERBOSE mode?

Reply:
Done.